internal/web/v1/countdown/countdown_get.go

```diff
@@ -2,27 +2,118 @@ package countdown
 
 import (
 	"context"
+	aulogging "github.com/StephanHCB/go-autumn-logging"
+	"github.com/eurofurence/reg-room-service/internal/config"
+	apierrors "github.com/eurofurence/reg-room-service/internal/errors"
+	"github.com/eurofurence/reg-room-service/internal/web/common"
+	"math"
 	"net/http"
+	"time"
 
 	modelsv1 "github.com/eurofurence/reg-room-service/internal/api/v1"
 )
 
+const (
+	mockTimeFormat    = "2006-01-02T15:04:05-07:00"
+	isoDateTimeFormat = "2006-01-02T15:04:05-07:00"
+	demoPublicSecret  = "[demo-secret]"
+	demoStaffSecret   = "[demo-staff-secret]"
+)
```

Comment:
const block in wrong file?

Reply:
Part of implementation details of the get endpoint. Would move out when another endpoint or verb is needed, for now let's keep things local to the file.